hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/resolver/MountTableResolver.java

```diff
@@ -460,9 +465,13 @@ public PathLocation getDestinationForPath(final String path)
         this.getLocCacheAccess().increment();
       }
       if (isTrashPath(path)) {
+        boolean useMountPointCreateTrashPath = conf.getBoolean(
```

Comment:
We usually want to expose all these things as configuration parameters but this might be too detailed.